src/characters/cbd-recipient.ts

```diff
@@ -46,55 +42,25 @@ export class ThresholdDecrypter {
   // Retrieve and decrypt ciphertext using provider and condition expression
   public async retrieveAndDecrypt(
     provider: ethers.providers.Provider,
-    signer: ethers.Signer,
     conditionExpr: ConditionExpression,
```

Comment:
You shouldn't need `conditionExpr` here. The condition is already in the threshold message kit object as part of the `acp`.